### ISSUE TITLE
Different icons and Launch Images assets for each flavor

### ISSUE THREAD

**Issue by 27comar** [AngeloAvv/flutter_flavorizr#4]
Hello,

First, thank you for this package which is extremely helpful to gain time when using flavors !

One issue I have and I don't seem to find an answer is : 

Is there a possibility to define a folder to take the different icons for the different flavors ? What I mean is, with the current version, it seems that all the flavors use the same launch icons and launch screens. 
Do I have to disable the "generateDummyAssets" for android and iOS and manually add the assets in the android folder and Xcode project for each flavor ? 

I hope you understand my question, if not just tell me and I can clarify ! 

Thank you for your time !

**Comment by AngeloAvv** [comment]
The process of customizing each flavor at this time is manual. The generateDummyAssets variable is used to copy the default icon set as a default placeholder, so you can customize them manually later.

**Comment by 27comar** [comment]
Okay, thank you for your answer !
I have another issue that I came across, I don't know if I should open another one or it's okay to ask it here but : 
Is there a way to NOT override the files that have already been generated using `flutter pub run flutter_flavorizr` ? What I mean by that is, if I customize the default icons (or the generated "main" file) but one day I add a new flavor, when I will run the command it will reset everything and put the default icons (and default main file) again on the flavors that were already created, am I right ?

**Comment by AngeloAvv** [comment]
flutter_flavorizr is meant to be run once and only once, after the creation of the flutter project. That's the reason why it overwrites some existing files.

If you want to reuse it, you can customize its process by defining a custom set of instructions. For example you can remove the *:dummyAssets, flutter:flavors (and/or) flutter:app instructions.
